qemu/tests/win_virtio_driver_installer_uninstall.py

```diff
@@ -83,21 +84,23 @@ def run(test, params, env):
     error_context.context("Check if all drivers are uninstalled.", test.log.info)
     # Wait a moment to check if drivers were uninstalled totally
     time.sleep(5)
+
     uninstalled_device = []
-    device_name_list = [
-        "VirtIO RNG Device",
-        "VirtIO Serial Driver",
-        "VirtIO Balloon Driver",
-        "QEMU PVPanic Device",
-        "VirtIO Input Driver",
-        "Red Hat VirtIO Ethernet Adapter",
-        "VirtIO FS Device",
-        "QEMU FwCfg Device",
+    # Filter out SCSI devices (cannot be uninstalled by installer)
+    need_uninstall_devices = [
+        name
+        for name in device_name_list
+        if not any(scsi in name for scsi in ["SCSI controller", "SCSI pass-through"])
```

Comment:
I think we can define a parameter here for unsupported device.Such like:
`unsupport_device_list = [
    "Red Hat VirtIO SCSI controller",
    "Red Hat VirtIO SCSI pass-through controller",
]
`

Reply:
I don't want to define the fixed variables in the py file. But I agree to add the no-need uninstallation device to a variable with logic.

Reply:
What is the difference between define a parameter for unsupported device and following code(if not any(scsi in name for scsi in ["SCSI controller", "SCSI pass-through"]))? Both of these methods are hard-coded.
The best method in my understanding should be define a parameter in cfg, define the parameter in py file also okay for me.